### PR TITLE
Add tabs to programme schedule

### PR DESCRIPTION
**Pull request by libuk**
screen shot 2018-01-07 at 12 34 31
screen shot 2018-01-07 at 12 32 56

### CODE REVIEW

File: static/css/custom.css [hunk 1]
```diff
@@ -11,7 +11,13 @@
     overflow-x: hidden !important;    
 }
 
+ul {
+    padding-left: 0;
+}
 
+li {
+    list-style-type: none;
```
Comment (libuk) [inline static/css/custom.css:19]: I added these global styles for `ul` and `li` as we were resetting them multiple times in the code. This allowed to remove some unnecessary styles.

Comment (DebsDee) [reply static/css/custom.css:19]: This is great, looks exactly how it should. The tabs aren't working yet but assume you're still working on them. Thank you.

Comment (libuk) [reply static/css/custom.css:19]: Hmm, it should be. What are you seeing?

Comment (DebsDee) [reply static/css/custom.css:19]: @libuk They seem to be working now, but weren't a second again. Weird.

Comment (libuk) [reply static/css/custom.css:19]: @DebsDee Maybe it was using an old copy of the JS file and needed a refresh or something

Comment (libuk) [reply static/css/custom.css:19]: @DebsDee How's it looking? All good, or you have some changes?

**Pull request merged**